enigma-cli/build.gradle

```diff
@@ -1,12 +1,15 @@
 plugins {
+    id 'application'
     id 'com.github.johnrengelman.shadow' version '5.2.0'
 }
 
 dependencies {
     implementation project(':enigma')
 }
 
-jar.manifest.attributes 'Main-Class': 'cuchaz.enigma.command.Main'
+mainClassName = 'cuchaz.enigma.command.Main'
```

Comment:
If you don't mind, maybe bump gradle and use the `mainClass` property from application plugin instead.